diagnostic.md

```diff
@@ -150,7 +187,10 @@ div {
 ```
 
 <!-- your answer starts here -->
+div one is going to be on the top right of the scree, with divs2 to div 3
+below div one aligned to teh left of the screen.
 
+The dive four and div five will be on a new line on the left of the screen...I think
```

Comment:
2 & 3 are on the same line as 1.


.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short: 2 3 1 4 5